vagrant.rb

```diff
@@ -99,6 +121,96 @@ def get_proxy()
   return $proxy
 end
 
+def configure_aws(config, ami, pf_name, pf_id, host_name, host_id, 
```

Comment:
It was easier to make a separate function than to merge this one and the 'virtualbox' configure.
Because of how Vagrantfile reading was done I used this difference to separate aws box and virtualbox ones
when looking for the server url.